package.json

```diff
@@ -13,16 +13,18 @@
   },
   "scripts": {
     "test": "npm run build && mocha && tsc && node ./ts-test/test.js && check-es3-syntax -p lib/ --kill",
-    "build": "babel es --out-dir lib",
+    "build": "babel es --out-dir lib && cp es/index.js.flow lib/",
```

Comment:
I don't think `es/index.js.flow` is need. When you import from `symbol-observable` package flow always resolves to main field.

Reply:
Why? Bundlers will resolve `module` field.

Reply:
Bundlers. Not Node.js. Check it just now:

`index.js`:
```js
import 'test'
```

`node_modules/test/package.json`:
```json
{
  "main": "cjs.js",
  "module": "esm.mjs"
}
```

`node_modules/test/cjs.js`:
```js
console.log('This is commonjs')
```

`node_modules/test/esm.mjs`:
```js
console.log('This is ES module')
```